### PR TITLE
Address -Woverloaded-virtual clang virtual and enable the warning

### PR DESCRIPTION
**Pull request by rouault**

### CODE REVIEW

File: src/app/devtools/networklogger/qgsnetworklogger.h [hunk 1]
```diff
@@ -76,6 +76,8 @@ class QgsNetworkLogger : public QAbstractItemModel
     QList< QAction * > actions( const QModelIndex &index, QObject *parent );
 
 
+    using QAbstractItemModel::removeRows;
+
     /**
```
Comment (nyalldawson) [inline src/app/devtools/networklogger/qgsnetworklogger.h:81]: This one could just be renamed -- it's not part of public api

**Review by nyalldawson — APPROVED** [review]

File: src/app/devtools/networklogger/qgsnetworklogger.h (hunk 1 above)
Comment (rouault) [reply src/app/devtools/networklogger/qgsnetworklogger.h:81]: any suggestion?

Comment (nyalldawson) [reply src/app/devtools/networklogger/qgsnetworklogger.h:81]: removeRequestRows?

Comment (rouault) [reply src/app/devtools/networklogger/qgsnetworklogger.h:78]: done

**Pull request merged**